src/Debugger/Impl/rtvs/R/packages.R

```diff
@@ -2,18 +2,30 @@
 # Licensed under the MIT License. See LICENSE in the project root for license information.
 
 matrix.as.lists <- function(matrix.data) {
-    res <- list()
-    dn <- dimnames(matrix.data)
+    res <- base::list()
```

Comment:
Just trying to preemptively prevent problems if user masks some of the functions we are using.  Do you think it's overkill?